### DISCUSSION TITLE
Issues that can likely be closed.

### DISCUSSION THREAD

**Discussion by Gazook89** [naturalcrit/homebrewery#1182] [General]
After commenting on a bunch of old issues advising they can likely be closed, I realized that it's probably better to find a way to group them into one spot rather than you guys getting a notification for each one (sorry about that).  So here is that list.

Saving:
----
- [x] #207: not saving, lost saves.  Fixed by google drive integration.  Does have some discussion by stolksdorf about user accounts and not wanting to pursue that.
- [x] #358: not saving, lost saves.  Fixed by google drive integration.
- [x] #393: not saving, lost saves.  Fixed by google drive integration.
- [x] #570: not saving, lost saves.  Fixed by google drive integration.
- [x] #573: not saving, lost saves.  Fixed by google drive integration.
- [x] #612: cannot find saved brews. Fixed by google drive integration.
- [x] #710: not saving, lost saves.  Fixed by google drive integration.  Questions about account recovery, though, which might mean we keep the issue open (although I suspect HB will never have this functionality).

PDF Issues:
----
- [x] #576: likely user error.
- [x] #613: tested this, could not reproduce.
- [x] #674: tested this, could not reproduce.
- [ ] #201:  Not closing. We need to verify that the problem is actually fixed (generating PDF on 140+ page brew crashes Chrome)
- [x] #324: tested this, could not reproduce.
- [x] #402: could not reproduce.
- [x] #423: user likely using firefox, no response to clarification request from calculuschild.
- [x] #440: not enough info to reproduce, no response to clarification request from calculuschild.
- [x] #448: not enough info to reproduce, and example given seems to be fine?  Assuming user figured it out.
- [x] #524: user error, many consumer grade printers don't print to the edge of a paper, and if they do, it is a setting they need to set.
- [ ] 

General Formatting Issues:
----
- [x] #126: fixed by wrapping classTables in their own div, completed by stolks.
- [x] #292: Stolksdorf requested additional info from user, no response.  Can't reproduce.
- [x] #297: can't reproduce.
- [ ] #302: Gonna keep this. We can add this to V3 styling.
- [x] #367: bad question, no source code given.
- [ ] #371: bad use of column breaks is sending most of the content to invisible columns off the page.
- [ ] #374: example code given isn't very helpful, hard to imagine what the problem was. I suspect just misuse of html/css.
- [ ] #389: no follow up response from user after clarification requested.  Not enough detail to reproduce.
- [x] #605: example code given to reproduce is not really applicable any longer, couldn't reproduce.

Rendering:
----
- [x] #273: 3 years old, haven't run into this issue or heard of it since I started using HB.
- [ ] #433: pretty sure this is just user error.
- [ ] #484: brew specific issue it seems on ipad. 3 years old, assuming not relevant any longer.

Meta Issues:
----
- [ ] #303: no log out button, is actually available on the naturalcrit front page.
- [ ] #450: user specific account issue, 3 years old, .....likely not relevant anymore.
- [ ] #480: user lost password, can be closed.
- [ ] #698: user lost password, can be closed.
- [ ] #703: user can't use login button..presumably not an issue 2 years later.

----

....going to hit the Start Discussion button just so I don't risk accidentally closing this tab and losing it all.  :)

**Comment by AlexeySachkov** [comment]
@Gazook89, I see that edits are possible, so I suggest to add a checkbox (`[ ]` for unchecked and `[X]` for checked one) into each item so we can track the status easily

**Comment by Gazook89** [comment]
Closing this discussion since every single one of the issues mentioned have been closed.
